src/main/java/com/learntechfree/ltf/entity/Article.java

```diff
@@ -0,0 +1,79 @@
+package com.learntechfree.ltf.entity;
+
+import lombok.*;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.Set;
+
+@Entity
+@Table(name = "articles",uniqueConstraints = {
+        @UniqueConstraint(columnNames = {"title","slug"})
+})
```

Comment:
In the future this will be confusing. So, lets give the constraint in their respective column rather then here and lets leave the table name as class name which will be `article` because in future we will might skip giving custom table name which results inconstancy in naming of table.

Reply:
Great 👍